app/src/main/java/org/schabi/newpipe/player/helper/SyncImageLoadingListener.java

```diff
@@ -0,0 +1,87 @@
+package org.schabi.newpipe.player.helper;
```

Comment:
Maybe you can move these classes related to frames and seeking into another subpackage, e.g. `org.schabi.newpipe.player.frames`